Free_Games.py

```diff
@@ -292,7 +297,7 @@ def wait_until_xpath_visible(xstr, wait_duration=10):
     
 def show_exception_and_exit(exc_type, exc_value, tb):
     traceback.print_exception(exc_type, exc_value, tb)
-    input("Press key to exit.")
+    print("Press key to exit.")

```

Comment:
again a toggle will be added but requiring user input inside a docker shell can be difficult.